src/docs/get-started/flutter-for/react-native-devs.md

```diff
@@ -206,27 +206,51 @@ objects to handle this.
 
 ```js
 // JavaScript
-_getIPAddress = () => {
-  const url="https://httpbin.org/ip";
-  return fetch(url)
-    .then(response => response.json())
-    .then(responseJson => {
-      console.log(responseJson.origin);
-    })
-    .catch(error => {
-      console.error(error);
-    });
-};
+class Example {
+  _getIPAddress() {
+    const url = 'https://httpbin.org/ip';
+    return fetch(url)
+      .then(response => response.json())
+      .then(responseJson => {
+        const ip = responseJson.origin;
+        return ip;
+      });
+  }
+}
+
+function main() {
+  const example = new Example();
+  example
+    ._getIPAddress()
+    .then(ip => console.log(ip))
+    .catch(error => console.error(error));
+}
+
+main();
 ```
 
 <!-- skip -->
 ```dart
 // Dart
-_getIPAddress() {
-  final url = 'https://httpbin.org/ip';
-  HttpRequest.request(url).then((value) {
-      print(json.decode(value.responseText)['origin']);
-  }).catchError((error) => print(error));
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+class Example {
+  Future<String> _getIPAddress() {
+    final url = 'https://httpbin.org/ip';
+    return http.get(url).then((response) {
+      String ip = jsonDecode(response.body)['origin'];
+      return ip;
+    });
+  }
+}
+
+main() {
+  final example = new Example();
+  example
+      ._getIPAddress()
+      .then((ip) => print(ip))
+      .catchError((error) => print(error));
```

Comment:
Wherever we're using non-DartPad-runnable examples, let's remove the "Try it out in DartPad" link just below the example